src/test/java/org/openrewrite/java/testing/jmockit/JMockitMockUpToMockitoTest.java

```diff
@@ -35,153 +34,232 @@ public void defaults(RecipeSpec spec) {
     void mockUpStaticMethodTest() {
         //language=java
         rewriteRun(
-          java(
-            """
-              import mockit.Mock;
-              import mockit.MockUp;
-              import static org.junit.Assert.assertEquals;
-              
-              class MockUpTest {
-                  void test() {
-                      new MockUp<MyClazz>() {
-              
-                          @Mock
-                          public int staticMethod() {
-                              return 1024;
-                          }
-              
-                          @Mock
-                          public int staticMethod(int v) {
-                              return 128;
-                          }
-                      };
-                      assertEquals(1024, MyClazz.staticMethod());
-                      assertEquals(128, MyClazz.staticMethod(0));
-                  }
-              
-                  public static class MyClazz {
-                      public static int staticMethod() {
-                          return 0;
-                      }
-              
-                      public static int staticMethod(int v) {
-                          return 1;
-                      }
-                  }
-              }
-              """,
-            """
-              import org.mockito.MockedStatic;
-              
-              import static org.junit.Assert.assertEquals;
-              import static org.mockito.Mockito.*;
-              
-              class MockUpTest {
-                  void test() {
-                      MockedStatic<MockUpTest.MyClazz> mockStaticMockUpTest_MyClazz = mockStatic(MockUpTest.MyClazz.class);
-                      mockStaticMockUpTest_MyClazz.when(() -> MockUpTest.MyClazz.staticMethod()).thenAnswer(invocation -> {
-                          return 1024;
-                      });
-                      mockStaticMockUpTest_MyClazz.when(() -> MockUpTest.MyClazz.staticMethod(anyInt())).thenAnswer(invocation -> {
-                          return 128;
-                      });
-                      assertEquals(1024, MyClazz.staticMethod());
-                      assertEquals(128, MyClazz.staticMethod(0));
-                      mockStaticMockUpTest_MyClazz.close();
-                  }
-              
-                  public static class MyClazz {
-                      public static int staticMethod() {
-                          return 0;
-                      }
-              
-                      public static int staticMethod(int v) {
-                          return 1;
-                      }
-                  }
-              }
-              """
-          )
+                java(
```

Comment:
To limit the indentation we use a .editorconfig file; would you mind double checking that's picked up correctly by your IDE?
https://github.com/openrewrite/rewrite-testing-frameworks/blob/e9497f1a56e075d5359a10b6356d580dc0ac5874/src/test/java/org/openrewrite/java/testing/.editorconfig#L3-L5

Reply:
Sure, I feel weird too.